knative-webhook/rockcraft.yaml

```diff
@@ -0,0 +1,71 @@
+# Based on ko image: https://github.com/knative/operator/tree/knative-v1.12.4/cmd/webhook
+name: knative-webhook
+summary: Knative webhook
+description: "Knative webhook"
+version: "1.12.4"
+license: Apache-2.0
+base: ubuntu@22.04
+platforms:
+    amd64:
+run-user: _daemon_
+
+environment:
+  # Required due to the go codebase relying on the OS Env being set
+  # See https://github.com/knative/operator/blob/knative-v1.12.4/pkg/reconciler/common/releases.go#L36
+  KO_DATA_PATH: "/var/run/ko"
+  # env identifies where to locate the SSL certificate file
+  SSL_CERT_FILE: "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
Is this file something we push to the container for the webhook service to find or is it generated elsewehere?

Reply:
the cert file is placed there by the `ca-certificates` package installed in L44, see also [this section of the ko spec](https://docs.google.com/document/d/1l_CrR_5U6NmDXpcoH2hkhVuoDzdpwFto6EwsylQ5vRw/edit?tab=t.0#heading=h.gcrneh498rpa)

Reply:
Gotcha! Thanks!